scripts/commitAndPush.sh

```diff
@@ -0,0 +1 @@
+yarn build; git add .; git commit -m "$1" -q; git push
```

Comment:
It's not very common to have this kind of script. What is it for exactly?

Reply:
Removed in latest